back/engines/commercial/smart_groups/spec/rules/participated_in_idea_status_spec.rb

```diff
@@ -242,24 +242,24 @@
         'nl-NL' => 'Reageerde niet op een idee met status in de prullenmand'
       })
       expect(participated_reacted_idea_in_idea_status_in_rule.description_multiloc).to eq({
-        'en' => 'Voted on an idea with one of the following statuses in the garbage can',
-        'fr-FR' => 'Voté pour une idée avec statut est un de dans la poubelle',
-        'nl-NL' => 'Stemde op een idee met één van de volgende statussen in de prullenmand'
+        'en' => 'Reacted to an idea with one of the following statuses in the garbage can',
+        'fr-FR' => 'Reacted to an idea with one of the following statuses dans la poubelle',
```

Comment:
Note, these tests will need updating again when we have the real translations back

Reply:
Alternatively, we could add translation files to the fixtures to avoid having to update the specs when translations change in Crowdin.